utils.py

```diff
@@ -0,0 +1,14 @@
+def get_input(split_lines=False, line_transform=None):
+    import inspect
+    from pathlib import Path
+
+    last_frame = inspect.stack()[1]
+    module = inspect.getmodule(last_frame[0])
+    input_path = Path(module.__file__).parent / "input.txt"
+    with open(input_path, "r") as f:
+        content = f.read().strip()
+        if split_lines:
+            content = content.splitlines()
+        if line_transform is not None:
+            content = [line_transform(line) for line in content]
```

Comment:
What do you think of this? I think maybe `line_transform` should imply `split_lines`

Not 100% sure about always calling `.strip()` on each line though 🤔 

```suggestion
        if split_lines or line_transform:
            content = content.splitlines()
	        if line_transform is not None:
	            content = [line_transform(line.strip()) for line in content]
```

Reply:
that's a good idea, one less args to pass. 

In terms of strip() I think it's better to make it configurable just in case...

Reply:
The `.strip()` can probably just be in `line_transform` tbh